### PR TITLE
Adds 2 new steal objectives: The Champion's Belt and Lamarr

### PR DESCRIPTION
**Pull request by swissloaf**
Two new steal objectives: steal the Champion's belt from the vault and Lamarr from the RD's office (the facehugger)

### Why is this change good for the game?

More variety for antag steal objectives

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

This PR literally just adds two new objectives: steal the vault gold belt and Lamarr the facehugger

### What should players be aware of when it comes to the changes your PR is implementing?

They are new steal objectives

### What general grouping does this PR fall under? 

Antag tweaking - new objectives

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here.

Filling out 50 sections on a PR sucks 

# Changelog

:cl:  
rscadd: Adds two new steal objectives: champion's belt and lamarr
/:cl:

### CODE REVIEW

**Comment by UselessTheremin** [comment]
already 2 objectives from the RD's office, and 2 objectives from the vault. Honestly this is getting to skewing the antag objectives tbh.

**Comment by alexkar598** [comment]
more objectives that security literally cant do anything about yayyyyyyyyy so much fun

**Comment by Marmio64** [comment]
Imagine upvoting your own PR, especially when everyone hates it.
